src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/CommonApiClient.java

```diff
@@ -88,6 +106,59 @@ public HttpResponse<String> sendNewSessionRequest(String sessionRequestBody)
         return sendHttpRequest(request);
     }
 
+    public HttpResponse<String> sendEventRequest(String sessionId)
+            throws IOException, InterruptedException {
+
+        URI eventsEndpointURI =
+                URI.create(
+                        TEST_RESOURCES_STACK
+                                + "events?partitionKey="
+                                + URLEncoder.encode("SESSION#", StandardCharsets.UTF_8)
+                                + sessionId
+                                + "&sortKey=TXMA");
```

Comment:
```suggestion
        final URI eventsEndpointURI = new URIBuilder(TEST_HARNESS_URL)
                .setPath("events")
                .addParameter("partitionKey", URLEncoder.encode("SESSION#", StandardCharsets.UTF_8) + sessionId)
                .addParameter("sortKey", "TXMA")
                .build();
```

Use the URL builder here